specification/datastore/DatastoreApis.ts

```diff
@@ -262,7 +262,7 @@ export const DatastoreApiSchemas = {
     }),
     result: z.object({}),
   },
-  'Datastore.createInMemoryFunction': {
+  'Datastore.createInMemoryRunner': {
```

Comment:
Why is this an api? Don't we always create a virtual table with results? Is this just for exploring the db?

Reply:
This is needed when you're querying locally. It must be created before it can be queried.